bundles/core/src/main/java/com/adobe/cq/commerce/core/cacheinvalidation/spi/CacheInvalidationContext.java

```diff
@@ -26,10 +26,10 @@
  * Context object containing all parameters needed for dispatcher cache invalidation.
  * This interface provides access to various components required for managing dispatcher cache invalidation
  * in the Commerce integration with AEM, including the page context, resource resolver,
- * attribute data, store path, and GraphQL client.
+ * invalidate type data, store path, and GraphQL client.
```

Comment:
Please use `invalidation type` instead of `invalidate type`. This occurs at multiple places in the last commit.
(Please note: `invalidate` is always a verb, the related noun is 'invalidation' see https://dictionary.cambridge.org/dictionary/english/invalidate)